packages/client/nginx.conf

```diff
@@ -61,6 +61,14 @@ server {
         proxy_pass http://countryconfig:3040/static/images/;
     }
 
+    location /api/ {
```

Comment:
Can we use /api/countryconfig here instead as I'm thinking the `/api/` would in the future refer to core's exposed endpoints